app/src/App.tsx

```diff
@@ -21,36 +24,48 @@ startMetadataLoad();
 function App() {
   const datasetStore = useDatasetStoreProvider();
   return (
-    <DatasetProvider value={datasetStore}>
-      <CssBaseline />
-      <div className={styles.App}>
-        <Router>
-          <AppBar position="static">
-            <Toolbar>
-              <Button className={styles.NavButton}>
-                <LinkWithStickyParams to="/">Home</LinkWithStickyParams>
-              </Button>
-              <Button className={styles.NavButton}>
-                <LinkWithStickyParams to="/datacatalog">
-                  Data Catalog
-                </LinkWithStickyParams>
-              </Button>
-              <Button className={styles.NavButton}>
-                <LinkWithStickyParams to="/exploredata">
-                  Explore the Data
-                </LinkWithStickyParams>
-              </Button>
-            </Toolbar>
-          </AppBar>
-          <Switch>
-            <Route path="/datacatalog" component={DataCatalogPage} />
-            <Route path="/exploredata" component={ExploreDataPage} />
-            <Route exact path="/" component={LandingPage} />
-            <Route component={NotFoundPage} />
-          </Switch>
-        </Router>
-      </div>
-    </DatasetProvider>
+    <ThemeProvider theme={MaterialTheme}>
+      <DatasetProvider value={datasetStore}>
+        <CssBaseline />
+        <div className={styles.App}>
+          <Router>
+            <AppBar position="static">
+              <Toolbar>
```

Comment:
I think it would be nice to pull the Toolbar out into another component (fine to leave it in the same file) to reduce indentation. I have a hard time parsing the html when it's nested this deep